### PR TITLE
Recipe interface

### PR DESCRIPTION
**Pull request by kyleoliveira**
@kathomas921 

Creating a pull request just for feedback, etc.

Don't merge, eh?

### CODE REVIEW

File: app/controllers/batches_controller.rb [hunk 1]
```diff
@@ -49,10 +51,14 @@ def create
       recipe.reload
       recipe.schedule.name.gsub!('Copy', "##{recipe.id}")
       recipe.name.gsub!('Copy', "##{recipe.id}")
+
       @batch.recipe = recipe
+        
+      if @batch.save(validate: false)
+        Recipe.update(recipe.id, :batch_id => @batch.id)
+        #why was this here? it undid all the recipe manipulation so batch.recipe was nil again.
+        #@batch.reload
```
Comment (kyleoliveira) [inline app/controllers/batches_controller.rb:60]: If you don't reload the object, the stuff on line 62 (formerly 57) would be nil. The point is that you save first, then refresh the object in memory from what was persisted.

File: app/controllers/batches_controller.rb [hunk 2]
```diff
@@ -19,7 +19,9 @@ def new
   end
 
   def assign_rhizomes
+    #what other option is there?
```
Comment (kyleoliveira) [inline app/controllers/batches_controller.rb:22]: See the BatchStateMachine class... it's not what you think.

File: app/views/schedules/_schedule.html.erb [hunk 3]
```diff
@@ -5,6 +5,13 @@
   </td>
   <td>
     <% if current_user.admin? %>
+    <%= link_to edit_schedule_path(schedule),
```
Comment (kyleoliveira) [inline app/views/schedules/_schedule.html.erb:8]: Been meaning to do this for a while...

File: app/controllers/batches_controller.rb (hunk 1 above)
Comment (kathomas921) [inline app/controllers/batches_controller.rb:60]: Ok, it seemed to be breaking something at some point, but perhaps I fixed a different bug at the same time and didn't realize it.

**Comment by kyleoliveira** [comment]
Closing this for now. Open a new PR when you're ready for review for your current Issue.

**Pull request closed without merging**